lib/select-builder.js

```diff
@@ -274,15 +119,21 @@ class SelectBuilder extends QueryBuilder {
     return this._addSelectClause('sum', field, alias);
   }
 
-  processFrom(from) {
+  _whereValueMapper(subquery) {
+    return typeof subquery === 'function'
+      ? subquery(new SelectBuilder(this.params, this.options))
+      : subquery;
+  }
+
+  _processFrom(from) {
     return mapJoinIterable(
       from,
-      f => (typeof f === 'string' ? f : `${f.table} AS ${f.alias}`),
+      f => (!f.alias ? String(f.table) : `${f.table} AS ${f.alias}`),
```

Comment:
Don't we use arrow-parens: always ?

Reply:
Oops, looks like i mixed up styles. Will update.